### DISCUSSION TITLE
Are there any options regarding depth of field?

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4676] [Q&A] [answered]
Title pretty much says it all. Are there any depth of field options that I'm not aware about?
If not, does anyone have any suggestions how I could do something about that? Or how it could be faked? 
I saw the grounded environment example and taught maybe I could do something with that?

Has anyone had encounters with this before? Any suggestions?

**Comment by elalish** [comment, marked as answer]
No, depth of field is pretty tricky to do well outside of a ray-tracer. The rasterizer approaches I've seen have all had pretty glaring artifacts. If you want the whole 3D model sharp and the background blurred, that's doable with a low-res or blurred skybox-image.

**Reply by danilokrulj** [reply]
Taught I would get a response like this but still had to double check :) thank you for the reply and suggestion
